frontend/src/css/imports.css

```diff
@@ -6743,18 +6744,18 @@ body,
   position: relative;
   height: calc(100% - 20px - 25px);
 }
-#worksheet .editable-field:hover {
+.editable-field:hover {
```

Comment:
When you hover on an editable field, the text moves slightly off around 1px. Can you fix this?